exercises/perfect-numbers/generator/generator.jl

```diff
@@ -0,0 +1,54 @@
+using Test
```

Comment:
this should be moved to the `generators/perfect-numbers.jl`, not `exercises/perfect-numbers/generator/generator.jl`. Sorry I didn't notice that earlier.